lib/xcov/model/report.rb

```diff
@@ -81,5 +88,19 @@ def self.excluded_targets
       excluded_targets
     end
 
+    def self.included_targets
```

Comment:
Mostly I was just trying to match the style and logic that existed before. I also had another PR open and was trying to minimize conflicts.

Im not much of a ruby dev, @nakiostudio do you have a style you prefer?